src/tests/class_tests/openms/source/AbsoluteQuantitationMethodFile_test.cpp

```diff
@@ -192,14 +208,62 @@ START_SECTION(void load(const String & filename, std::vector<AbsoluteQuantitatio
   transformation_model_params = aqm_list[7].getTransformationModelParams();
   TEST_REAL_SIMILAR(transformation_model_params.getValue("slope"), 1.0);
   TEST_REAL_SIMILAR(transformation_model_params.getValue("intercept"), 2.0);
+
+  // The following input file doesn't have the headers: component_name, llod
```

Comment:
Are you checking for a missing `transformation_model_param_` column as well?

Reply:
Yes, for example here: https://github.com/biosustain/OpenMS/pull/110/files/96dc372ab4cedc953abb3beecd66325ca98566c6..364567b3d1d0265751ca48019f5b2562b2e2d332#diff-966d9b27049c1f1c5958431fecea06c7R242
No value was present within the file, so a default value is set.